plugins/logitech-tap/fu-logitech-tap-sensor-device.c

```diff
@@ -370,22 +349,12 @@ fu_logitech_tap_sensor_device_init(FuLogitechTapSensorDevice *self)
 				 FU_UDEV_DEVICE_FLAG_OPEN_READ | FU_UDEV_DEVICE_FLAG_OPEN_WRITE |
 				     FU_UDEV_DEVICE_FLAG_OPEN_NONBLOCK |
 				     FU_UDEV_DEVICE_FLAG_IOCTL_RETRY);
-	/* setup device identifier so plugin can distinguish device during composite_cleanup */
-	fu_device_add_private_flag(FU_DEVICE(self), FU_LOGITECH_TAP_DEVICE_FLAG_TYPE_SENSOR);
-}
-
-static void
-fu_logitech_tap_sensor_device_finalize(GObject *object)
-{
-	G_OBJECT_CLASS(fu_logitech_tap_sensor_device_parent_class)->finalize(object);
 }
 
 static void
 fu_logitech_tap_sensor_device_class_init(FuLogitechTapSensorDeviceClass *klass)
 {
-	GObjectClass *object_class = G_OBJECT_CLASS(klass);
 	FuDeviceClass *klass_device = FU_DEVICE_CLASS(klass);
-	object_class->finalize = fu_logitech_tap_sensor_device_finalize;
```

Comment:
If finalize not required, remove it from fu-logitech-tap-hdmi-device.c as well!

Reply:
Ack, done.